src/main/scala/com/fulcrumgenomics/umi/DuplexConsensusCaller.scala

```diff
@@ -191,6 +223,19 @@ class DuplexConsensusCaller(override val readNamePrefix: String,
     }
   }
 
+  /** Split records into those that should make a single-end consensus read, first of pair consensus read,
+    * and second of pair consensus read, respectively.  For [[DuplexConsensusCaller]], no fragment reads should be
+    * given to this method, so none will be returned.
+    */
+  override protected def subGroupRecords(records: Seq[SamRecord]): (Seq[SamRecord], Seq[SamRecord], Seq[SamRecord]) = {
```

Comment:
Can you comment that this is presumably overridden this way for performance reasons?  Otherwise it would seem much simpler to just let the super class do this.